doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -0,0 +1,139 @@
+
+Release Walkthrough
+===================
+
+Building the release
+--------------------
+
+Checkout the branch for the release, make sure it is up to date, and clean the
+repository::
+
+    $ git checkout maintenance/1.11.x
+    $ git pull upstream maintenance/1.11.x
+    $ git submodule update
+    $ git clean -xdf
+
+Look at the git log to get the hash of the last commit in the release, then
+check it out::
+
+    $ git log
+    $ git checkout e57701a40
+
+Edit pavement.py and setup.py as detailed in HOWTO_RELEASE::
+
+    $ gvim pavement.py setup.py
+    $ git commit -a -m"REL: NumPy 1.11.2rc1 release."
+
+Sanity check::
+
+    $ python runtests.py -m "full"
+    $ python3 runtests.py -m "full"
+
+Tag it, install, and build the source distribution files for sourceforge. The
+install is needed for the documentation which is built using the current
+installed numpy::
+
+    $ git tag -s v1.11.2rc1
+    $ python setup.py install --user
+    $ paver sdist
+    $ paver write_release_and_log
+    $ paver pdf
```

Comment:
I think we should move away from SourceForge, so this is not needed anymore then. Now that we don't distribute .exe installers there's really no use for SF anymore, and it has a pretty bad reputation by now (deservedly).

What is missing is the procedure to upload pdf and html docs to docs.scipy.org, which would be good to add instead.